Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
+members = [
+    "cli",
```

Comment:
Is it not a problem that `cli/` and `cli/Cargo.toml` don't exist (yet)? In any case it might be confusing, so as long as `cli/` is not available, I would not include it here.